h2o-py/h2o/backend/cluster.py

```diff
@@ -249,6 +249,10 @@ def get_status_details(self):
         table = H2OTwoDimTable(cell_values=node_table, col_header=keys, row_header=keys)
         return table
 
+    def get_dkv(self):
+        info = h2o.api("GET /3/Cloud")
+        return [node['dkv'] for node in info._props['nodes']]
```

Comment:
you seem to be iterating over all the nodes, however, given the transient nature of the field - it will only be defined on the requesting node

moreover - this API is to dangerous to be directly exposed here

I would imagine this would be its own API, hidden from users